packages/aragon-wrapper/src/utils/index.test.js

```diff
@@ -23,3 +23,18 @@ test('should enhance an object to lookup eth addresses easier', async (t) => {
   t.is(permissions['0x0000000000000000000000000000000000000abC'], undefined)
   t.is(permissions['0x0000000000000000000000000000000000000ABc'], undefined)
 })
+
+test('should initialise the proxy with its keys in lowercase', async (t) => {
```

Comment:
Perhaps we could rename this test to

`should allow the proxy to be initialized with an object containing any cased keys`

And add a few different variations (mixed case, all lower case).